src/seismometer/controls/explore.py

```diff
@@ -636,7 +633,10 @@ def __init__(self, title: str, option_widget: ValueWidget, plot_function: Callab
         self.option_widget.observe(self._on_option_change, "value")
         self.update_plot_widget.on_click(self._on_plot_button_click)
         self.update_plot_widget.on_toggle_code(self._on_toggle_code)
-        self.update_plot_widget.trigger()
+
+        plot_args, plot_kwargs = self.generate_plot_args()
```

Comment:
Are update_plot_widget calls changing state or just attaching callbacks and other setup?

If it's the latter, should some of these new lines be consolidated with the group above?
ex. ~634 + ~639 both "generate_plot_code()", ~634 and ~641 both set "current_plot_code"